src/components/DailyConfigComponent.tsx

```diff
@@ -18,41 +19,51 @@ import localStorageSet from "roamjs-components/util/localStorageSet";
 import nanoid from "nanoid";
 
 const DailyConfig = () => {
-  // TODO refresh config so debug panel shows up to date info
   const config = useMemo(getDailyConfig, []);
-  const [disabled, setDisabled] = useState(!config.enabled);
+  const [localConfig, setLocalConfig] = useState(config);
+  const {
+    enabled,
+    device,
+    time: setRunTime,
+    "workflow name": workflowName,
+    "last-run": lastRun,
+    "next-run": configNextRun,
+  } = localConfig;
   const [onlyOnThisDevice, setOnlyOnThisDevice] = useState(
-    !!config.device && config.device === localStorageGet("device")
+    !!device && device === localStorageGet("device")
```

Comment:
Id move `localStorageGet("device")` to `getDailyConfig` defaults

Reply:
I couldn't get localStorageGet to work in zDailyConfig.

I tried
`currentDevice: localStorageGet("device"),`
`currentDevice: z.string().optional().default(""),`
and
`const currentDevice = localStorageGet("device");`
``currentDevice: currentDevice,``